version/version.go

```diff
@@ -15,7 +15,7 @@ const (
 	Project = "ts-bridge"
 
 	// Ver is the current version of this project.
-	Ver = "0.1.1"
+	Ver = "0.1.2"
```

Comment:
Nit: As per [Semantic Versioning](https://semver.org/) guidelines patch version is reserved for fixes, not added functionality, so I would increment the minor version, i.e. make it `0.2.0`